_usage/query_planning.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: default
+title: Reviewing a Query Plan
+nav_order: 8
+---
+
+## Reviewing a Query Plan
+
+Babelfish version 2.1.0 includes support for SET statements that allow you to review estimated and actual (executed) query plans. This allows you to use the TDS port to identify and refine slow-performing queries. The functions can display an query plan for:
+
+- `SELECT`, `INSERT`, `UPDATE`, and `DELETE` statements
+- Nested procedures, multiple statements in a batch
+- Variable declarations (for example, `DECLARE @t TABLE (a int, b int);`)
+
+The SET statements *do not* support query plan retrieval from functions, control flows, and cursors.
+
+You can use a SET statement on the TDS port to turn on/off the following functions: 
+
+- SET BABELFISH_STATISTICS PROFILE {ON|OFF} to display the query plan used to execute a statement. The command implements the behavior of the [PostgreSQL EXPLAIN ANALYZE statement] (https://www.postgresql.org/docs/current/using-explain.html#USING-EXPLAIN-ANALYZE).
+    
+- SET BABELFISH_SHOWPLAN_ALL {ON|OFF} to display estimated execution plans for a statement without performing the command. The command implements the behavior of the [PostgreSQL EXPLAIN statement](https://www.postgresql.org/docs/current/using-explain.html#USING-EXPLAIN-BASICS).
+
+Please note that you should not enable both SET statements at the same time.
+
+The following example turns on query planning, displays the query results and profile, and then turns off query planning:
+
+```
+SET BABELFISH_STATISTICS PROFILE ON
+GO
+
+SELECT CategoryID, CategoryName FROM dbo.categories ORDER BY 1 DESC
+GO
+
+CategoryID  CategoryName
+----------- ---------------
+8           Seafood
+7           Produce
+6           Meat/Poultry
+5           Grains/Cereals
+4           Dairy Products
+3           Confections
+2           Condiments
+1           Beverages
+(8 rows affected)
+QUERY PLAN
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Query Text: select CategoryID, CategoryName from dbo.categories order by 1 desc
+Sort  (cost=45.59..47.17 rows=630 width=36) (actual rows=8 loops=1)
+  Sort Key: categoryid DESC NULLS LAST
+  Sort Method: quicksort  Memory: 25kB
+  ->  Seq Scan on categories  (cost=0.00..16.30 rows=630 width=36) (actual rows=8 loops=1)
+Completion time: 2022-06-13T10:18:53.3661572-07:00
+
+SET BABELFISH_STATISTICS PROFILE OFF
+GO
+```
+
+For example, the following command sequence returns an estimated cost to execute the SELECT statement and then turns off query planning:
```

Comment:
it doe snot turn off query planning (= the query optimizer). It turns off displaying of the query plan.